grails-app/services/org/pih/warehouse/core/OrganizationService.groovy

```diff
@@ -82,6 +81,9 @@ class OrganizationService {
                     ilike("description", "${params.q}%")
                 }
             }
+            if (params.name) {
+                ilike("name", "%${params.name}%")
```

Comment:
Can't you do it using the above part for it (pass the name as a `q`)?

Reply:
Sure I could but I think that in case of getting organizations for dropdown I think that we should only get those with matching name and not code or description.

Reply:
@jmiranda How do you see it?

Reply:
@DominikaHatala Where is params.name coming from? I only see params.q in the form?

Reply:
Nevermind, just realized this was probably being sent as a parameter from a React component. I think we probably want it to work the same as the search form on the Org list page.

Reply:
So yeah let's pass in params.q and then if we get specific requirements from kelsey about how name or description should match then we can implement those separately.